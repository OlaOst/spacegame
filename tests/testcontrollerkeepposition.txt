test.position = [0.0, 0.0]
test.control = KeepPosition
test.positionToKeep = [1.0, 1.0]
