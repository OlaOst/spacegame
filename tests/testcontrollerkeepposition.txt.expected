test.position = [1, 1]
test.control = KeepPosition
test.positionToKeep = [1.0, 1.0]
